modules/tinymce/changelog.txt

```diff
@@ -2,6 +2,7 @@ Version 5.6.0 (TBD)
     Added new `format_empty_lines` setting to control if empty lines are formatted in a ranged selection #TINY-6483
     Added new user interface `enable` and `disable` methods #TINY-6397
     Added new `closest` formatter API to get the closest matching selection format from a set of formats. #TINY-6479
+    Fixed `readonly` mode to allow hyperlinks to be clickable #TINY-6248
```

Comment:
I wonder if this should be `Changed`? As it was always intended that it was being blocked.